front/components/editor/extensions/EmojiExtension.tsx

```diff
@@ -0,0 +1,22 @@
+import Emoji, { emojis, shortcodeToEmoji } from "@tiptap/extension-emoji";
+
+import { createEmojiSuggestion } from "@app/components/editor/input_bar/emojiSuggestion";
+
+export const EmojiExtension = Emoji.extend({
+  renderMarkdown: (node) => {
+    const emojiItem = shortcodeToEmoji(node.attrs?.name, emojis);
+
+    return emojiItem?.emoji ?? `:${node.attrs?.name}:`;
+  },
+}).configure({
+  // Enable emoticon conversion (e.g., <3 → ❤️, :) → 😊)
```

Comment:
Would be nice to fix this heart display at some point 🥺

<img width="98" height="90" alt="Image" src="https://github.com/user-attachments/assets/c6acae0f-db06-4b6c-a588-59d293e7f703" />